test/robot.js

```diff
@@ -19,6 +20,14 @@ describe('Robot', function () {
       }
     }
 
+    event2 = {
+      event: 'arrayTest',
+      payload: {
+        action: 'bar',
+        installation: {id: 2}
+      }
+    }
+
```

Comment:
Since this is only used in one test, I would just move it into the tests it's used.